src/metabase/util/malli/humanize.cljc

```diff
@@ -0,0 +1,71 @@
+(ns metabase.util.malli.humanize
+  (:require
+   [malli.error :as me]))
+
+(defn- resolve-error
+  "This is the same behavior as what [[malli.error/humanize]] does to resolve errors."
```

Comment:
An inline description would be nice. At least an example. Does it produce a pair of `[path message]`?